pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -137,6 +137,8 @@ type ShootSpec struct {
 	// This field is immutable.
 	// +optional
 	SchedulerName *string `json:"schedulerName,omitempty" protobuf:"bytes,21,opt,name=schedulerName"`
+	// CloudProfile is a reference to a CloudProfile or a NamespacedCloudProfile.
```

Comment:
```suggestion
	// CloudProfile contains a reference to a CloudProfile or a NamespacedCloudProfile.
```

Nit: To follow the existing doc structure.